strategies/changed/src/test/java/org/arquillian/smart/testing/vcs/git/GitFetcherTest.java

```diff
@@ -0,0 +1,106 @@
+package org.arquillian.smart.testing.vcs.git;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.OpenOption;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.Set;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.diff.DiffEntry;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import static org.arquillian.smart.testing.vcs.git.GitRepositoryUnpacker.unpackRepository;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GitFetcherTest {
```

Comment:
Then something is odd with commits / github diff - but that's fine. Just curious